pkg/components/exporter/exporter.go

```diff
@@ -257,7 +257,7 @@ func NewConfigMap(d components.Detail, k *v1alpha1.KeplerInternal) *corev1.Confi
 		"EXPOSE_HW_COUNTER_METRICS":  "true",
 		"EXPOSE_IRQ_COUNTER_METRICS": "true",
 		"EXPOSE_KUBELET_METRICS":     "true",
-		"EXPOSE_CGROUP_METRICS":      "true",
+		"EXPOSE_CGROUP_METRICS":      "false",
```

Comment:
https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/pkg/components/exporter/exporter.go#L620 Ahem ahem :p

Reply:
Also, I think we can completely remove this from the configmap
cc: @marceloamaral ?

Reply:
I think by default Kepler takes value of `ENABLE_EBPF_CGROUPID` as [true](https://github.com/sustainable-computing-io/kepler/blob/29d0b00f2362ac28d14b887af9a3eb6140f281b0/pkg/config/config.go#L70) and `EXPOSE_CGROUP_METRICS` as [true](https://github.com/sustainable-computing-io/kepler/blob/29d0b00f2362ac28d14b887af9a3eb6140f281b0/pkg/config/config.go#L77)